FeedStoreChallenge/CoreDataFeedStore.swift

```diff
@@ -29,14 +29,48 @@ public final class CoreDataFeedStore: FeedStore {
 	}
 
 	public func retrieve(completion: @escaping RetrievalCompletion) {
-		fatalError("Must be implemented")
+		let context = context
+		context.perform {
```

Comment:
Just FYI, there's another way of capturing `context` inside the block so that `self` is not captured. One can use block's capture list. The following suggestion does 100% the same but 1 line shorter

```suggestion
		context.perform { [context] in
```